keras_nlp/models/bert/bert_models.py

```diff
@@ -226,13 +227,43 @@ def presets(cls):
         return copy.deepcopy(backbone_presets)
 
     @classmethod
+    @format_docstring(names=", ".join(backbone_presets))
```

Comment:
This is much more readable, nice. It may have some trouble scaling to many custom string inserts, but for now I don't think we have this problem.